### PR TITLE
Add comment field to service accounts

### PR DESCRIPTION
**Pull request by vadmeste**
## Description

This PR will add Comment field to Credentials for
all type of accounts, regular users, STS and service 
accounts.

However, currently, it only gives users support to 
add/edit comments of service accounts.

## Motivation and Context
Add comments to service accounts, so users can remember what they are used for.

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Optimization (provides speedup with no functional changes)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [ ] Fixes a regression (If yes, please add `commit-id` or `PR #` here)
- [ ] Unit tests added/updated
- [ ] Internal documentation updated
- [x] Create a documentation update request [here](https://github.com/minio/docs/issues/new?label=doc-change,title=Doc+Updated+Needed+For+PR+github.com%2fminio%2fminio%2fpull%2fNNNNN)

### CODE REVIEW

**Comment by minio-trusted** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-large-bucket.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-erasure.sh | :heavy_check_mark: |
| mint-compress-encrypt-dist-erasure.sh | :heavy_check_mark: |
| mint-pools.sh | :heavy_check_mark: |
| mint-dist-erasure.sh | :x: [more...](#4490ee2d-ce62-4783-b3e0-d95de7893eca) |
<a name="4490ee2d-ce62-4783-b3e0-d95de7893eca"></a>
### 16380-ff2288c/mint-dist-erasure.sh.log:
```
Running with
SERVER_ENDPOINT:      15.15.15.2:30637
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0
RUN_ON_FAIL:          0

To get logs, run 'docker cp 24a40122216c:/mint/log /tmp/mint-logs'

(1/14) Running aws-sdk-go tests ... done in 1 seconds
(2/14) Running aws-sdk-java tests ... done in 2 seconds
(3/14) Running aws-sdk-php tests ... done in 42 seconds
(4/14) Running aws-sdk-ruby tests ... done in 7 seconds
(5/14) Running awscli tests ... done in 1 minutes and 0 seconds
(6/14) Running healthcheck tests ... done in 0 seconds
(7/14) Running mc tests ... done in 16 seconds
(8/14) Running minio-go tests ... done in 44 seconds
(9/14) Running minio-java tests ... FAILED in 27 seconds
{
  "name": "minio-java",
  "function": "addUser()",
  "duration": 420,
  "status": "FAIL",
  "error": "java.lang.RuntimeException: Request failed with response: {\"Code\":\"InternalError\",\"Message\":\"We encountered an internal error, please try again. (Server not initialized, please try again)\",\"Resource\":\"/minio/admin/v3/add-user\",\"RequestId\":\"1738A101C7C90B4A\",\"HostId\":\"daec108d-1438-40af-8825-4ec91688c20c\"}\n >>> [io.minio.admin.MinioAdminClient.execute(MinioAdminClient.java:206), io.minio.admin.MinioAdminClient.addUser(MinioAdminClient.java:237), TestMinioAdminClient.addUser(TestMinioAdminClient.java:47), TestMinioAdminClient.runAdminTests(TestMinioAdminClient.java:160), FunctionalTest.runTests(FunctionalTest.java:3782), FunctionalTest.runEndpointTests(FunctionalTest.java:3881), FunctionalTest.main(FunctionalTest.java:4002)]"
}
(9/14) Running minio-js tests ... done in 50 seconds
(10/14) Running minio-py tests ... done in 1 minutes and 40 seconds
(11/14) Running s3cmd tests ... done in 15 seconds
(12/14) Running s3select tests ... done in 3 seconds
(13/14) Running versioning tests ... done in 3 minutes and 5 seconds

Executed 13 out of 14 tests successfully.
```
Deleting image on docker hub
Deleting image locally

File: cmd/admin-handlers-users.go [hunk 1]
```diff
@@ -954,6 +959,8 @@ func (a adminAPIHandlers) InfoServiceAccount(w http.ResponseWriter, r *http.Requ
 		return
 	}
 
+	fmt.Println("info", svcAccount)
```
Comment (klauspost) [inline cmd/admin-handlers-users.go]:
```suggestion
```

File: cmd/admin-handlers-users.go [hunk 2]
```diff
@@ -21,6 +21,7 @@ import (
 	"bytes"
 	"encoding/json"
 	"errors"
+	"fmt"
```
Comment (klauspost) [inline cmd/admin-handlers-users.go]:
```suggestion
```

**Review by klauspost — COMMENTED** [review]
Debug print left.

File: cmd/admin-handlers-users.go (hunk 1 above)
Comment (vadmeste) [reply cmd/admin-handlers-users.go]: Thanks!

**Review by harshavardhana — APPROVED** [review]

File: go.mod [hunk 3]
```diff
@@ -232,3 +232,5 @@ require (
 	gopkg.in/square/go-jose.v2 v2.6.0 // indirect
 	gopkg.in/yaml.v3 v3.0.1 // indirect
 )
+
+replace github.com/minio/madmin-go/v2 => github.com/vadmeste/madmin-go/v2 v2.0.0-20230106172704-da31e43c554a
```
Comment (harshavardhana) [inline go.mod]: madmin-go is merged, you can update the dependency.

**Review by harshavardhana — CHANGES_REQUESTED** [review]

File: go.mod (hunk 3 above)
Comment (vadmeste) [reply go.mod]: > madmin-go is merged, you can update the dependency.

Done

**Comment by minio-trusted** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-fs.sh | :heavy_check_mark: |
| mint-erasure.sh | :heavy_check_mark: |
| mint-dist-erasure.sh | :heavy_check_mark: |
| mint-compress-encrypt-dist-erasure.sh | :heavy_check_mark: |
| mint-pools.sh | :heavy_check_mark: |
| mint-large-bucket.sh | :x: [more...](#19fb6b46-33e4-4ccd-9d98-5c766ab4026d) |
<a name="19fb6b46-33e4-4ccd-9d98-5c766ab4026d"></a>
### 16380-fb56eda/mint-large-bucket.sh.log:
```
Running with
SERVER_ENDPOINT:      15.15.15.5:30013
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0
RUN_ON_FAIL:          0

To get logs, run 'docker cp b212425417b4:/mint/log /tmp/mint-logs'

(1/14) Running aws-sdk-go tests ... done in 2 seconds
(2/14) Running aws-sdk-java tests ... done in 1 seconds
(3/14) Running aws-sdk-php tests ... done in 42 seconds
(4/14) Running aws-sdk-ruby tests ... done in 8 seconds
(5/14) Running awscli tests ... done in 1 minutes and 0 seconds
(6/14) Running healthcheck tests ... done in 0 seconds
(7/14) Running mc tests ... done in 16 seconds
(8/14) Running minio-go tests ... done in 47 seconds
(9/14) Running minio-java tests ... FAILED in 31 seconds
{
  "name": "minio-java",
  "function": "addUser()",
  "duration": 415,
  "status": "FAIL",
  "error": "java.lang.RuntimeException: Request failed with response: {\"Code\":\"InternalError\",\"Message\":\"We encountered an internal error, please try again. (Server not initialized, please try again)\",\"Resource\":\"/minio/admin/v3/add-user\",\"RequestId\":\"1738EC0FC43F164A\",\"HostId\":\"64ba38ca-2c70-40eb-83b1-1c66d37f5c5f\"}\n >>> [io.minio.admin.MinioAdminClient.execute(MinioAdminClient.java:206), io.minio.admin.MinioAdminClient.addUser(MinioAdminClient.java:237), TestMinioAdminClient.addUser(TestMinioAdminClient.java:47), TestMinioAdminClient.runAdminTests(TestMinioAdminClient.java:160), FunctionalTest.runTests(FunctionalTest.java:3782), FunctionalTest.runEndpointTests(FunctionalTest.java:3881), FunctionalTest.main(FunctionalTest.java:4002)]"
}
(9/14) Running minio-js tests ... done in 49 seconds
(10/14) Running minio-py tests ... done in 1 minutes and 49 seconds
(11/14) Running s3cmd tests ... done in 16 seconds
(12/14) Running s3select tests ... done in 4 seconds
(13/14) Running versioning tests ... done in 3 minutes and 6 seconds

Executed 13 out of 14 tests successfully.
```
Deleting image on docker hub
Deleting image locally

**Review by harshavardhana — APPROVED** [review]

**Comment by harshavardhana** [comment]
PTAL @klauspost

**Review by klauspost — APPROVED** [review]
LGTM

**Pull request merged**